CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.49.0 / 2021-07-06
+
+* [CHANGE] Flag "storage.tsdb.no-lockfile" will now default to false. #4066
+* [CHANGE] Remove `app.kubernetes.io/version` label selector from prometheus and alertmanager statefulsets. #4093
+* [FEATURE] Added web tls configuration support for prometheus. #4025
+* [FEATURE] Add proxy_url support for Probes. #4043
+* [ENHANCEMENT] Set proper build context in version package. #4019
+* [ENHANCEMENT] Added publishing images on GitHub Container Registry. #4060
+* [ENHANCEMENT] Make failure to update StatefulSet more readable. #4100
```

Comment:
It can also be skipped (IMHO changes to log messages aren't worth a mention in general).

```suggestion
* [ENHANCEMENT] Better log message when the operator needs to recreate a StatefulSet. #4100
```